carsus/io/nuclear/nndc.py

```diff
@@ -0,0 +1,87 @@
+import pandas as pd
+import os
+
+DECAY_DATA_SOURCE_DIR = "/home/syzygy/Desktop/TARDIS/carsus-data-nndc-main/csv"
```

Comment:
It would be great to set a more sensible default. I think Carsus has a default download directory you can use.